### PR TITLE
install certificates

### PR DESCRIPTION
**Pull request by jburel**
this commit was missed from the previous branch.
Certificates needs to be installed for ice

cc @sbesson

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ ARG JAVAVER=${JAVAVER:-openjdk1.8}
 # Download and run omero-install.
 ENV OMERO_INSTALL /tmp/omero-install/linux
 
-RUN yum install -y git \
+RUN yum install -y git certifactes \
```
Comment (joshmoore) [inline Dockerfile:15]: Typo?

**Comment by sbesson** [comment]
From https://github.com/ome/devslave-c7-docker/runs/5040576688?check_suite_focus=true ```No package certifactes available.```

**Comment by joshmoore** [comment]
But why didn't it fail? (That's why I removed my comment ;) )

**Comment by sbesson** [comment]
Looks like `yum install` has a lenient behavior by default. There are options like `skip_missing_names_on_install` that would fail if the package was not found

**Comment by jburel** [comment]
```
--> Running transaction check
---> Package ca-certificates.noarch 0:2015.2.6-73.el7 will be updated
---> Package ca-certificates.noarch 0:2021.2.50-72.el7_9 will be an update
```

**Comment by jburel** [comment]
Merging and tagging to 0.7.1

**Comment by joshmoore** [comment]
> Looks like `yum install` has a lenient behavior by default...

Reading through https://serverfault.com/questions/694942/yum-should-error-when-a-package-is-not-available, I assume the easiest/best idiom/workaround is to have each `yum install` separately:

```
RUN yum install -y foo \
 && yum install -y bar
```

**Pull request merged**